app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,101 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/vHeaderImage"
+        android:layout_width="match_parent"
+        android:layout_height="204dp"
+        android:layout_marginBottom="68dp"
+        android:scaleType="fitXY"
+        android:background="@drawable/login_cover"
+        android:src="@drawable/wolox_colors_gradient"
+        app:layout_constraintTop_toTopOf="parent"></ImageView>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@+id/vLoginButton"
+        app:layout_constraintTop_toTopOf="@id/vHeaderImage">
+
+        <ImageView
+            android:id="@+id/vWoloxLogoImage"
+            android:layout_width="match_parent"
+            android:layout_height="145dp"
+            app:srcCompat="@drawable/wolox_logo_large"></ImageView>
+    </RelativeLayout>
+
+    <EditText
+        android:id="@+id/vFirstNameInput"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="90dp"
+        android:layout_marginRight="16dp"
+        android:hint="@string/fragment_login_firstname"
+        android:inputType="text"
+        app:layout_constraintTop_toBottomOf="@id/vHeaderImage"></EditText>
+
+    <EditText
+        android:id="@+id/vLastNameInput"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:hint="@string/fragment_login_lastname"
+        android:inputType="text"
+        app:layout_constraintTop_toBottomOf="@id/vFirstNameInput"></EditText>
+
+    <Button
+        android:id="@+id/vLoginButton"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="16dp"
+        android:background="@color/colorWhite"
+        android:text="@string/fragment_login_button_log_in"
+        app:layout_constraintTop_toBottomOf="@id/vLastNameInput"></Button>
+
+    <Button
+        android:id="@+id/vSignUpButton"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="16dp"
+        android:background="@color/colorBlueButton"
+        android:text="@string/fragment_login_button_sign_up"
+        android:textColor="@color/colorWhite"
+        app:layout_constraintTop_toBottomOf="@+id/vLoginButton"></Button>
+
+    <LinearLayout
```

Comment:
Since you are using constraint layout, you dont need to use a horizontal linear layout to implement this. Try yo do it without this layout